llvm/unittests/Support/KnownBitsTest.cpp

```diff
@@ -752,8 +765,10 @@ TEST(KnownBitsTest, CommonBitsSet) {
           HasCommonBitsSet |= N1.intersects(N2);
         });
       });
-      EXPECT_EQ(!HasCommonBitsSet,
-                KnownBits::haveNoCommonBitsSet(Known1, Known2));
+      if (!Known1.hasConflict() && !Known2.hasConflict()) {
```

Comment:
Same here.

Reply:
I meant that you could save some time here by testing for conflict as the first thing in the loop body, before the nested ForeachNumInKnownBits loops.